pkg/tenant/db/members.go

```diff
@@ -22,29 +22,35 @@ const (
 )
 
 type Member struct {
-	OrgID        ulid.ULID    `msgpack:"org_id"`
-	ID           ulid.ULID    `msgpack:"id"`
-	Email        string       `msgpack:"email"`
-	Name         string       `msgpack:"name"`
-	Role         string       `msgpack:"role"`
-	Status       MemberStatus `msgpack:"status"`
-	Created      time.Time    `msgpack:"created"`
-	Modified     time.Time    `msgpack:"modified"`
-	DateAdded    time.Time    `msgpack:"date_added"`
-	LastActivity time.Time    `msgpack:"last_activity"`
+	OrgID        ulid.ULID `msgpack:"org_id"`
+	ID           ulid.ULID `msgpack:"id"`
+	Email        string    `msgpack:"email"`
+	Name         string    `msgpack:"name"`
+	OrgName      string    `msgpack:"org_name"`
+	OrgDomain    string    `msgpack:"org_domain"`
+	UsageType    string    `msgpack:"usage_type"`
+	Interests    []string  `msgpack:"interests"`
+	Role         string    `msgpack:"role"`
```

Comment:
Is there is a reason to desync this from Quarterdeck? The role can be obtained from the claims … if it is important to have this information; we'll need to ensure this is kept up to date wrt Quarterdeck (e.g. having a middleware that ensures the role is correct from Quarterdeck claims).